### ISSUE TITLE
Error on login

### ISSUE THREAD

**Issue by frazzmark** [afwang/ogame-android-app#18]
When I try to login the app says "Error loading events!" and a toast with "Check user name and password". Does the app work with Italian website?

**Comment by afwang** [comment]
No, the app currently does not support the game running on a different language other than English (the .org site). Support for multiple languages is a long-term goal, however.

**Comment by frazzmark** [comment]
Could logs be useful to that purpose?
Il 05/ago/2015 18:12 afwang <notifications@github.com> ha scritto:No, the app currently does not support the game running on a different language other than English (the .org site). Support for multiple languages is a long-term goal, however.

**Comment by afwang** [comment]
For that purpose, we need a copy of the HTML of the page, or at least a translation of the ship names and mission names, as exactly represented in the game (upper and lower case matters).
